options/options.c

```diff
@@ -1000,6 +1001,9 @@ static const struct MPOpts mp_default_opts = {
     .lua_load_select = true,
     .lua_load_positioning = true,
     .lua_load_commands = true,
+#ifndef _WIN32
```

Comment:
This looks iffy. What about macOS, or other possible implementations of context menu?

Reply:
What is iffy? When the macOS context menu is implemented this needs to be update to `#if !defined(_WIN32) && !defined(HAVE_COCOA)`

Reply:
>When the macOS context menu is implemented

macOS has its own context menu implemented already.

Reply:
It just reduces the number of threads running.